packages/hs-react-ui/src/components/InteractionFeedback/InteractionFeedback.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useState } from 'react';
+import { animated, useTransition } from 'react-spring';
+import styled, { StyledComponentBase } from 'styled-components';
+import shortid from 'shortid';
+import colors from '../../enums/colors';
+import useResizeObserver from 'use-resize-observer/polyfilled';
+
+// TODO Reduce amount of any/ts-ignore done here
+
+const Circle = styled.svg`
+  pointer-events: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+`;
+
+const Container = styled(animated.div)`
+  position: relative;
+`;
+
+enum feedbackTypes {
```

Comment:
It may be worthwhile to move this to the enums directory and import it to keep consistent with other components?

Reply:
Do these feedback types do anything at the moment? I figure giving them the ability to directly set transition config control negates the need for presets.. Though I could see it being a thing in the future perhaps